src/lib/util/version.ml

```diff
@@ -29,7 +29,7 @@
 (* WARNING: a "cut" is performed on the following file in the Makefile.
    DO NOT CHANGE its format *)
 
-let _version="dev"
+let _version="models"
```

Comment:
revert to `dev` (and make note that this string must be changed when preparing a new release).